core/helpers/EEH_Sideloader.helper.php

```diff
@@ -71,9 +71,9 @@ public function set_upload_to($upload_to_folder)
     {
         $this->_upload_to = $upload_to_folder;
     }
-    public function set_upload_from($upload_from_folder)
+    public function set_upload_from($upload_from)
     {
-        $this->_upload_from_folder = $upload_from_folder;
+        $this->_upload_from = $upload_from;
```

Comment:
might want to keep the parameter name as `$upload_from_folder` since it is actually a bit more descriptive since it tells anybody looking at this that it represents a folder name. Might be even better to rename the parameter to `$folder_name` so that the a call to the function becomes even more descriptive, ie: `set_upload_from($folder_name)`, whereas keeping it as is (`set_upload_from($upload_from)`) leaves it kind of cryptic as to what is expected.

So one extra job that will be required of you now that you have touched this file (ya opened a can of worms boy!) is to go through and bring it's formatting and phpdocs up to date. This primarily means adding phpdocs to all of the methods that don't have them since it looks like the formatting is already decent for the most part.

Reply:
I think we spoke about this in slack a while back but `$_upload_from` would actually be the full path for the file you want to download, so `$upload_from_folder` is incorrect and `set_upload_from($folder_name)` even more so (although I completely understand why you would suggest it).

Personally I find most of the 'upload_' parameters used in the sideloader a little confusing as we aren't referring to an upload in a single step, we want to download_from into a temp directory, then then 'upload'/move to the location.

So to me something like this makes more sense:

```
$sideloader_args = array(
    '_upload_to'     => EE_PLUGIN_DIR_PATH . 'languages/',
    '_download_from'   => 'https://github.com/eventespresso/languages-ee4/blob/master/event_espresso.pot?raw=true',
    '_new_file_name' => 'event_espresso.pot',
);
```

Obviously any `download_` properties would have their own setter/getters etc.

Anyway, I digress :)

Sorry for the huge delay on this, I set it aside until I set up PHPStorm.... then never actually set it up until now.

Reply:
Oh, and something I wasn't 100% here is `@since $VID:$` for this method.

It simply won't work at all up until this change so I'm assuming I use the above rather than when it was actually added?

Reply:
no you can skip adding the `@since` tag since that is used to indicate a new method added
any methods that do not have that tag can simply be considered legacy code